prelude/slang-torch-prelude.h

```diff
@@ -38,6 +38,23 @@
 #   include <stdint.h>
 #endif // SLANG_LLVM
 
+#if defined(_MSC_VER)
+#   define SLANG_PRELUDE_SHARED_LIB_EXPORT __declspec(dllexport)
+#else
+#   define SLANG_PRELUDE_SHARED_LIB_EXPORT __attribute__((__visibility__("default")))
```

Comment:
What's the reason to remove attribute(dllexport) ? It's there for windows compatibility it seems, but that seems reasonable if you built windows binary with clang/gcc. Or if you were building under cygwin etc.

Reply:
Yeah, this is added here to get things to compile for now. I don't know what the perfect definition would be here.